bench/00_operators/operators.cu

```diff
@@ -19,9 +20,10 @@ void vector_add(nvbench::state &state, nvbench::type_list<ValueType>)
 
   tensor_t<ValueType, 1> xv{{x_len}};
   tensor_t<ValueType, 1> xv2{{x_len}};
+
   xv.PrefetchDevice(0);
```

Comment:
I should have told you this, but can you please remove all the Prefetch calls like this one too?

Reply:
What should they be replaced by, if anything?

Reply:
Nothing. They can be removed